packages/vsce/src/commands/getFilterPlexResources.ts

```diff
@@ -32,81 +68,67 @@ export function getFilterPlexResources(tree: CICSTree, treeview: TreeView<any>)
     } else if (selection[selection.length - 1] && selection[selection.length - 1] instanceof CICSRegionsContainer) {
       chosenNode = selection[selection.length - 1];
     } else {
-      window.showErrorMessage("No 'Regions' node selected");
+      window.showErrorMessage(l10n.t("No 'Regions' node selected"));
       return;
     }
     const plex = chosenNode.getParent();
     await treeview.reveal(chosenNode, { expand: true });
     const plexProfile = plex.getProfile();
-    let resourceToFilter: any;
-    if (plexProfile.profile.regionName && plexProfile.profile.cicsPlex) {
-      resourceToFilter = await window.showQuickPick(["Programs", "Local Transactions", "Local Files", "Tasks", "Libraries"]);
-    } else {
-      resourceToFilter = await window.showQuickPick(["Regions", "Programs", "Local Transactions", "Local Files", "Tasks", "Libraries"]);
+
+    const RESOURCE_CHOICES_BASE = [
+      { id: "Programs", label: l10n.t("Programs") },
```

Comment:
Working back from getCmciKeyForLabel, i think we should avoid this.

Can we list the metas for the resources we want to enable, e.g. `ProgramMeta`, then we can use `<metafile>.resourceName` as the ID and `<metafile>.humanReadableNameSingular` as the display value.